awesome_owl/static/src/todo_list/todo_list.js

```diff
@@ -0,0 +1,30 @@
+import {Component, useState} from "@odoo/owl"
+import { TodoItem } from "./todo_item";
+import { useAutofocus } from "../utils";
+
+export class TodoList extends Component{
+    static template = "awesome_owl.TodoList";
+    static components = {TodoItem}
+    setup(){
+        this.id=0;
+        this.todos = useState([]);
+        useAutofocus("input")
+    }
+    addTodo(ev){
+        if(ev.keyCode == 13 && ev.target.value != ""){
+            this.todos.push({
+                id : this.id++,
+                description : ev.target.value,
+                isCompleted : false
+            })
+            ev.target.value = "";
+        }
+    }
+    toggleTodo(todoid){
```

Comment:
```suggestion
    toggleTodo(todoId){
```
Camel casing?
Please follow this [guide](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#variables)